### DISCUSSION TITLE
Upgrade to 1.10 version

### DISCUSSION THREAD

**Discussion by eandrienko** [akvorado/akvorado#1224] [Q&A] [answered]
Hello!
Accordingly to release notes for 1.10 - "New installations should also get **better compression and performance** from the
main table, due to a change to the primary key used for this table".
Does this mean I need to reinstall my current 1.8 install to get this performance boost or going standard upgrade way is to be enough? "New installations" slightly confusing me.

**Comment by vincentbernat** [comment, marked as answer]
The easy way is to destroy the clickhouse container and volume, but you loose your data. Otherwise see #1223.
